assets/csidriveroperators/vsphere/15_prometheusrules.yaml

```diff
@@ -28,3 +28,19 @@ spec:
             including its Deployment, DaemonSet and CSIDriver objects and OpenShift will
             automatically install a supported version of the CSI driver.
           message: "An unsupported version of vSphere CSI driver installation detected."
+      - alert: VSphereTopologyMisconfiguration
+        # Both source="infrastructure" and source="clustercsidriver" must be above 0
+        expr:  min(vsphere_topology_tags{source=~"infrastructure|clustercsidriver"}) > 0
```

Comment:
I feel like a query ex-plainer is in order, because it is a clever query!

> min() works by taking minimum over both `infrastructure` and `clustercsidriver` dimensions and only when both are 1, then value will be greater than  0 and an alert will be triggered.

I think  we should add this comment.

Reply:
rephrased your comment a bit, PTAL